ipfsapi/http.py

```diff
@@ -243,3 +241,7 @@ def session(self):
         yield
         self._session.close()
         self._session = None
+
+
+class HTTPClient(HTTPClientBase):
```

Comment:
Why declare this class? Just to have a concrete type inheriting from `HTTPClientBase`?